massa-grpc/src/public.rs

```diff
@@ -398,7 +398,7 @@ pub(crate) fn get_stakers(
         });
 
     // Get the current cycle and slot.
-    let now: MassaTime = MassaTime::now()?;
```

Comment:
can the error variant corresponding to this "?" be removed as well ?

Reply:
Done